### ISSUE TITLE
List open windows matching given search term

### ISSUE THREAD

**Issue by vitaliis** [linuxmint/cinnamon-spices-applets#5382]
### Applet name and maintainer

Cinnamenu@json @fredcw

### What would you like to see?

Display a section with open windows matching the given search term. Like a section with emojis appearing when there are matches.

An example can be seen in "Windows Quick List (windows-quick-list@cinnamon.org)" applet. It just does not have a hotkey to open the list and navigate it. And no filter capabilities.

It can be optionally enabled (to be displayed or not) as per configuration in Cinnamenu.
Enabled or disabled by default.

**Comment by fredcw** [comment]
It would probably be much easier to add a hotkey option to the window-quick-list applet. Not sure I want to add this to cinnamenu unless there was a big demand for it. Another option is to make a search provider so that it would work in any of the menu applets. I'll have a think about it.

**Comment by vitaliis** [comment]
Yes, but then also an input field with search (filter) capability. Which is more or less already present in Cinnamenu. 

So implementing it there seems more involved and more complex.

**Comment by vitaliis** [comment]
About a search provider I haven't heard yet. Just checked what you mean by this. And if I am not mistaken, it would mix up everything in one group. E.g. results of windows, matching applications, files and all other search results. Showing up all under the same group intertwined with each other. Unlike, for example emojis, which has its own group for displaying matching results.
